src/diffusers/loaders.py

```diff
@@ -1802,6 +1802,10 @@ def from_single_file(cls, pretrained_model_link_or_path, **kwargs):
             tokenizer ([`~transformers.CLIPTokenizer`], *optional*, defaults to `None`):
                 An instance of `CLIPTokenizer` to use. If this parameter is `None`, the function loads a new instance
                 of `CLIPTokenizer` by itself if needed.
+	        mislljson_dir (`str`, *optional*, defaults to 'DIFFUSERS_CACHE'):
```

Comment:
`tryharderfindmodel` ?

Reply:
i think that part is removed?but yeah the core is to try find the vae and etc.